example_school_library_decorator/rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    person.rentals << self
+  end
```

Comment:
- [ ] Also here you should add the attr_accessor for **person**, I do see only **@date & @book**.